src/components/elements/side-tab.js

```diff
@@ -161,6 +175,18 @@ const SideTab = ({ children, has_hash_routing, is_sticky }) => {
         <StyledSideTab>
             <TabList is_sticky={is_sticky}>
                 <Desktop>
+                    {tabHeader ? (
+                        <Header
```

Comment:
can we use !!tabHeader &&  <Header>... ? since the second option of the ternary is undefined